### PR TITLE
Initial integration of frontend with backend

### PR DESCRIPTION
**Pull request by potatoeggy**
The following have been implemented:

 - The lowest square now changes colour when the column is clicked.
 - Debug coordinates in the previous commit were removed.
 - A rudimentary win check has been implemented so we can call this initial integration.

### CODE REVIEW

File: src/GameWindow.java [hunk 1]
```diff
@@ -84,9 +84,14 @@ public void mouseExited(MouseEvent event) { // get rid of highlight when mouse l
...
 
 	public void actionPerformed(ActionEvent event) { // this is going to contain a lot of stuff too
-		// this is temporary since the board class should be updated to handle column only things
-		int x = (Integer) (((JButton) event.getSource()).getClientProperty("x"));
-		int y = (Integer) (((JButton) event.getSource()).getClientProperty("y"));
-		eventHandler.getBoard().addChip(x, y, eventHandler.getCurrentPlayer());
+		int column = (Integer) (((JButton) event.getSource()).getClientProperty("y"));
+		int row = eventHandler.getBoard().addChip(column, eventHandler.getCurrentPlayer());
+		if (row != -1) {
+			buttonGrid[row+1][column].setBackground(eventHandler.getCurrentPlayer() == 1 ? Color.RED : Color.YELLOW); // reminder: work around board zero limitations by using playerIndex - 1 when getting names
```
Comment (vincentjguo) [inline src/GameWindow.java]: You don't have to do this but you could disable the button once it is clicked - just a quality of life improvement

**Review by vincentjguo — APPROVED** [review]

File: src/GameWindow.java (hunk 1 above)
Comment (potatoeggy) [reply src/GameWindow.java]: Yeah sure I'll do that

**Comment by potatoeggy** [comment]
O boi that was hard

Review?

**Comment by vincentjguo** [comment]
the chips aren't showing up for me

**Comment by potatoeggy** [comment]
Make sure resources is accessible to the compiled class. This usually means you need to manually compile the program or move the images to \<workingdirectoryofclass\>/resources.

**Comment by vincentjguo** [comment]
I think it is accessible by the program because there are no errors being generated. It just doesn't show up.

**Comment by potatoeggy** [comment]
If the images cannot be found an error is not thrown. This happens to me too when I compile in VSCode.

**Comment by potatoeggy** [comment]
I've added lots of checks, and now it should yell at you if it detects an error.

**Comment by vincentjguo** [comment]
yep apparently it can't find the file

**Comment by vincentjguo** [comment]
where did you place the resource file? in src?

**Comment by potatoeggy** [comment]
yep

**Comment by vincentjguo** [comment]
apparently Intellij checks for a file outside of src

**Comment by vincentjguo** [comment]
it still doesn't work tho, the error isn't showing but the pictures aren't either

**Comment by potatoeggy** [comment]
That's because the compiled .class file is probably dumped in some temporary directory.

**Comment by vincentjguo** [comment]
ok now it works

**Comment by vincentjguo** [comment]
but only with src/resource/..

**Comment by potatoeggy** [comment]
Should be fixed now

**Comment by vincentjguo** [comment]
yepp

**Review by vincentjguo — APPROVED** [review]

**Pull request merged**